src/js/hyperschedule.js

```diff
@@ -1669,21 +1669,66 @@ function downloadPDF()
           course.courseName,
           width - 12,
         );
+        const courseLocationLines = pdf.splitTextToSize(
+          slot.location,
+          width - 12,
+        );
+        const courseInstructorsLines = pdf.splitTextToSize(
+          course.faculty.map(fullName => fullName.split(",")[0]).join(", "),
```

Comment:
The code to make a list of last names could be pulled off into a helper function